src/pages/Home.js

```diff
@@ -35,10 +36,18 @@ const SearchWrapper = styled.div`
   background: #216968;
 `
 
+const FilterWrapper = styled.div`
+  display: grid;
+  min-height: 250px;
+  height: 100%;
+  align-content: start;
+  gap: 5px;
+  padding: 25px 15px;
+`
+
 const FullScreenGrid = styled(Grid)`
   height: 100vh;
   width: 100vw;
-  overflow: hidden; //TODO fix map overflow
```

Comment:
was `overflow: hidden` causing issues?

Reply:
sorry it was my bad removed thinking that this is invalid syntax while was looking for what was wrong with the formatting, it turned out to be something else, shared in the chat.

Reply:
updated